lib/flatpak-installation.c

```diff
@@ -1121,6 +1121,31 @@ get_write_progress (guint outstanding_writes)
   return (guint) (3 / (gdouble) outstanding_writes);
 }
 
+static gboolean
+app_contains_extra_data (const char *app_id)
+{
+  /* HACK: This is a temporary solution until the detached metadata work is ready */
+  const char *extra_data_apps[] =
```

Comment:
Do you need to hardcode the list here? Do you not have a way from inside flatpak to determine that the app had extra data?